app/src/main/java/me/quadphase/qpdex/DetailedPokemonActivity.java

```diff
@@ -196,6 +197,8 @@ public void preventFurtherExecution(){
     private TextView evolutionTab;
     private TextView alternatesTab;
 
+    private final int MAX_NUMBER_OF_TYPES = 17;
```

Comment:
This variable shouldn't be defined in a UI class and I also don't see it being used. It should be deleted.